ceph-installer-rpm/config/ceph-installer-rpm.yml

```diff
@@ -0,0 +1,39 @@
+- job:
+    name: ceph-installer-rpm
+    node: 'centos7 && x86_64 && small'
+    project-type: freestyle
+    defaults: global
+    disabled: false
+    display-name: 'ceph-installer: RPMs'
+    description: 'Build RPMs for every ceph-installer Git branch'
+    concurrent: true
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph
+    discard-old-builds: true
+    logrotate:
+      daysToKeep: 1
+      numToKeep: 10
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+
+    triggers:
+      - github
+
+    scm:
+      - git:
```

Comment:
I'd like to have the ability to build ceph-ansible and ceph-installer RPMs automatically from any branch in order to make it easier to test changes while keeping master stable